transformation/src/it/test-files/outputs/gene_association/part-1.json

```diff
@@ -1,10 +1,10 @@
-{"content":"{\"@FullName\":\"atlastin GTPase 1\",\"Location\":{\"CytogeneticLocation\":{\"$\":\"14q22.1\"},\"SequenceLocation\":[{\"@Assembly\":\"GRCh38\",\"@AssemblyAccessionVersion\":\"GCF_000001405.38\",\"@AssemblyStatus\":\"current\",\"@Chr\":\"14\",\"@Accession\":\"NC_000014.9\",\"@start\":\"50533082\",\"@stop\":\"50633068\",\"@display_start\":\"50533082\",\"@display_stop\":\"50633068\",\"@Strand\":\"+\"},{\"@Assembly\":\"GRCh37\",\"@AssemblyAccessionVersion\":\"GCF_000001405.25\",\"@AssemblyStatus\":\"previous\",\"@Chr\":\"14\",\"@Accession\":\"NC_000014.8\",\"@start\":\"50999799\",\"@stop\":\"51099785\",\"@display_start\":\"50999799\",\"@display_stop\":\"51099785\",\"@Strand\":\"+\"}]},\"OMIM\":{\"$\":\"606439\"}}","gene_id":"51062","variation_id":"4352","relationship_type":"within single gene","source":"submitted"}
```

Comment:
For the 2nd reviewer: It's tough to see, but the changes here are from `@FullName` disappearing from the `content` field. This is expected now that it's being properly extracted into a field in the `gene` table.